Doc/whatsnew/3.13.rst

```diff
@@ -324,6 +324,10 @@ Removed
   certificates for you.
   (Contributed by Victor Stinner in :gh:`105382`.)
 
+* Remove deprecated ``webbrowser.MacOSXOSAScript._name`` attribute.
+  Use :attr:`webbrowser.MacOSXOSAScript.name` attribute instead.
```

Comment:
Sorry would you like to link to this attribute? https://docs.python.org/3.13/library/webbrowser.html#webbrowser.name

Reply:
Done!